### PR TITLE
IR-4256 Include assets in model component export path

### PR DESCRIPTION
**Pull request by AidanCaruso**
## References
closes #_insert number here_

### CODE REVIEW

File: packages/ui/src/components/editor/properties/model/index.tsx [hunk 1]
```diff
@@ -108,7 +108,7 @@ export const ModelNodeEditor: EditorComponentType = (props) => {
       return
     }
     exporting.set(true)
-    const fileName = `${srcPath.value}.${exportType.value}`
+    const fileName = `assets/${srcPath.value}.${exportType.value}`
```
Comment (HexaField) [inline packages/ui/src/components/editor/properties/model/index.tsx:111]: we either need another argument to `exportRelativeGLTF` or to split the filename when passed into the `File` constructor as it shouldn't have the path in it

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/ui/src/components/editor/properties/model/index.tsx (hunk 1 above)
Comment (AidanCaruso) [reply packages/ui/src/components/editor/properties/model/index.tsx:111]: Yes. My original solution was to modify exportRelativeGLTF to break this up.

This broke prefab export, which I had forgotten also references exportRelativeGLTF, so it was reverted. Didn't want to make core changes close to release so I mirrored what we're already doing for prefabs and included assets in the relative path argument to make this not error out. I'll look at breaking it up again (in a way that doesn't bork prefabs).

**Review by dinomut1 — CHANGES_REQUESTED** [review]
This is fixed on dev. Instead of this just fix the regex splitting done in the model node editor. See line 87 of this file on dev: https://github.com/ir-engine/ir-engine/blob/dev/packages/ui/src/components/editor/properties/model/index.tsx

File: packages/ui/src/components/editor/properties/model/index.tsx (hunk 1 above)
Comment (dinomut1) [inline packages/ui/src/components/editor/properties/model/index.tsx:111]: We don't want to hard-code exports into the "assets" folder, to preserve flexibility with folder structure

**Review by dinomut1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**